### DISCUSSION TITLE
Is there a legend for the issue/changelog prefixes?

### DISCUSSION THREAD

**Discussion by aaronmader** [pyproj4/pyproj#1297] [Q&A] [answered]
The [changelog](https://pyproj4.github.io/pyproj/stable/history.html) and [issue tracker](https://github.com/pyproj4/pyproj/issues) are using subject line prefixes, but I'm not sure what all of them mean. 

- DEP: Short for Dependency. An issue / change regarding a dependency.
- WHL: Short for .. wheel? Something about python wheels?
- ENH: Short for .. ?? Seems to be for new features?
- PERF: Short for Performance. For performance related items.
- CLN: Short for Clean. For code cleanup / clarity.
- REF: Short for ... Reference? Maybe this is documentation related?
- BUG: For bugs. Code isn't doing what it should.
- MNT: Short for... ??

I'm mostly reaching out to ask because I'm working through the process of updating my dependencies on a project, and I'm reviewing your changelog to quickly determine whether there are any breaking changes, or changes in behaviour that I need to be aware of. And I found some of these prefixes to be unclear.

You do appear to be following semantic versioning, so I know I should be safe to pop up a minor version.. but it's always worth a quick review. :)

**Comment by snowman2** [comment, marked as answer]
Probably a good idea to add that to the [CONTRIBUTING](https://github.com/pyproj4/pyproj/blob/main/CONTRIBUTING.md) guide sometime.

You have it mostly right. Here is the filled in version:

-   DEP: Short for Dependency. An issue / change regarding a dependency.
-   WHL: Short for Wheel. Change related to python wheels.
-   ENH: Short for Enhancement.  For new features.
-   PERF: Short for Performance. For performance related items.
-   CLN: Short for Clean. For code cleanup / clarity.
-   REF: Short for Refactor. For code refactoring related changes.
-   BUG: For Bugs. Code isn't doing what it should.
-   MNT: Short for Maintenance.

**Reply by aaronmader** [reply]
Thank you @snowman2!
